### ISSUE TITLE
Base URL & Reverse Proxy "Your session has ended"

### ISSUE THREAD

**Issue by jamisonhyatt** [deluan/navidrome#146]
Hey, great project and thanks for the base path feature! I'm having some issues with it and not sure what is going on.  I have multiple other services running in this same setup, so I think it has something to do with the UI in Navidrome.

I have Navidrome running with base URL `/navidrome` behind an Apache httpd Reverse proxy that requires basic authentication by users, as configured below.  Initial user creation page loads fine, and adding the first user works. However immediately upon login the UI landing page is shown and I'm immediately kicked out with a message `Your session has ended, please reconnect.` 

I enabled debug logs and there weren't any errors from this action.

Looking at the network requests in the browser, the login POST gets a 200, then the fetch to 
https://{apache host}/navidrome/app/api/artist?_end=15&_order=ASC& receives a 401. 

I don't know how much work it would be, but I think an option to disable all authentication within navidrome for scenarios where users plan on running it behind something that handles authentication would be useful (and avoid this situation, I think.)

FWIW everything works when going directly to the host:port, so it definitely has to do with some combination of behaviors between the reverse proxy and the web UI.
```
  <Location /navidrome>
                ProxyPass http://127.0.0.1:4533/navidrome retry=1
                ProxyPassReverse http://127.0.0.1:4533/navidrome
                ProxyErrorOverride Off
                ProxyPreserveHost On
                AuthType Digest
                AuthName "Welcome"
                AuthDigestProvider file
                AuthUserFile /etc/apache2/passwords
                Require valid-user
        </Location>
```

**Comment by deluan** [comment]
Thanks for trying Navidrome!

Not sure what could be causing it, I'm using it behind a Caddy proxy and it is working fine. I'll try to reproduce this on my side, using Apache.

Re: disabling authentication, I feel it is more important to invest time in #141. Not saying it won't be done, but right now there are other priorities... Can you create a separated issue to track this?

**Comment by deluan** [comment]
It may be a conflict of Apache's digest authentication and Navidrome's own authentication: both use the `Authentication` header, so Apache may be overriding ND's header.

Can you try something? Can you disable Apache authentication for Navidrome?

**Comment by deluan** [comment]
I'll release this fix as part of 0.13.1, later tonight
